src/Vehicle/Vehicle.cc

```diff
@@ -913,6 +913,7 @@ void Vehicle::_handleSysStatus(mavlink_message_t& message)
         _batteryFactGroup.voltage()->setRawValue(VehicleBatteryFactGroup::_voltageUnavailable);
     } else {
         _batteryFactGroup.voltage()->setRawValue((double)sysStatus.voltage_battery / 1000.0);
+        _batteryFactGroup.currentWatts()->setRawValue(_batteryFactGroup.voltage()->rawValue().toFloat()*_batteryFactGroup.voltage()->rawValue().toFloat());
```

Comment:
I think you made a mistake here W != V^2

Reply:
Also, I think it makes more sense to use the incoming sysStatus properties instead of the facts for the calculation.

Reply:
Oh sorry, the second should be current, my fault.